sa/sa.go

```diff
@@ -893,13 +893,10 @@ func (ssa *SQLStorageAuthority) AddCertificate(
 	regID int64,
 	ocspResponse []byte,
 	issued *time.Time) (string, error) {
-	// NOTE(@cpu): Historically `AddCertificate` was hardcoded to set the added
-	// `core.Certificate`'s `Issued` field to the current time. If the `issued`
-	// parameter is nil then default to using now as the issued time to preserve
-	// this historic default.
+	// This should not happen - the gRPC layer enforces that issued != nil. If it
+	// does occur it should be considered an internal server error
 	if issued == nil {
-		now := ssa.clk.Now()
-		issued = &now
+		return "", berrors.InternalServerError("AddCertificate received nil issued argument")
```

Comment:
>  (in my mind I imagine there is a proper way to do this without breaking deployability but maybe I'm wrong).

If you don't know how this could be done without breaking deployability (I'm not sure either) I'd prefer to leave it as a pointer type.

> Our general rule seems to be that we trust the gRPC layer to check for things being nil and if that happens here something has gone seriously wrong elsewhere (and maybe a panic in that case is better than a 500?).

Ok! I'll remove the nil check in the SA impl :+1: